wiki/Rules/Code_of_conduct_for_modding_and_mapping/en.md

```diff
@@ -4,58 +4,28 @@ The **code of conduct for modding and mapping** is a set of rules and guidelines
-- When pointing out "issues", inspect each thoroughly and consider exactly what it is about the particular "thing" you dislike. Then, try to offer a potential solution or improvement and consider exactly how that suggestion will improve the map. If necessary, try multiple suggestions to see which work best — the more help you can offer the mapper, the better.
-
-Suggestions themselves should ideally consist of four main parts, though based on the experience of the mapper and/or your personal relationship to them, you may want to be more lenient:
-
-1. **A statement of the issue itself and where it can be found** - including a timestamp is a huge help so that the mapper can find the part in question quickly.
-2. **A brief explanation of why this causes a problem** - this is essential in convincing a mapper to even consider your suggestion. Mappers are much more receptive to criticism and change when it's accompanied by clear, understandable and valid reasoning.
-3. **One or more alternative suggestions, accompanied by explanations of how they address the problem(s)** - much like the above, explanation is key here. Even if you can convince the mapper that their current work can be improved, what makes your suggestion better than theirs, or anyone else's?
-4. **The wording (or even language) used should take the mapper into consideration.** Are they a fluent English speaker? If not, stick to simple and easy-to-understand language. If you both speak the same language, by all means write in that language, but try to append an English summary of your mods and mod replies if possible, to ensure everyone stays on the same page. If necessary, support your suggestions with screenshots, illustrations, or even short video clips to help get your point across.
-
-There are any number of ways to present your mod post, so feel free to experiment, but try to keep the visuals simple and formatted in a way that it is easy for the mapper and others to work with.
+- **The goal of modding is to help a mapper improve their work.** If you're not interested in helping people, steer clear of the discussion.
+- **Consider what you're posting, why you're posting it, and who you're posting it to.** This will ensure your best intentions are conveyed and your post is easy to understand.
+- **Be approchable.** Discussions are open to all users. Everyone is welcome to participate, including those who are new to the community. Be patient, understanding, and provide as much explanation as necessary so that everyone can learn and improve more quickly.
+- **Treat others as you wish to be treated.** People are more receptive to kind, helpful suggestions than hostile ones. Stay on topic and keep away from personal attacks, hateful comments, and misplaced jokes to avoid derailing the thread.
```

Comment:
Could write `[...] avoid derailing the thread and risking a [silence](/wiki/Silence).` If you did the things mentioned, that could happen.

Reply:
I guess this is already achieved by the **Report bad behaviour** bulletpoint. The alternative would be to move it or echo it in the introduction with a catch-all sentence along the lines of *"Not adhering to the following code of conduct may result in a silence."*, or something of the sorts.

Reply:
Will resolve for now. Reports and potential punishments are already covered in the article (See **Report bad behaviour** bulletpoint and the last sentence in the introduction). The article for Silences is covered at the introduction.